bluemira/equilibria/flux_surfaces.py

```diff
@@ -408,7 +408,7 @@ def reset_direction(coords):
 
         # Pick the first intersection, travelling from the o_point outwards
         deltas = x_inter - o_point.x
-        arg_inter = np.argmax(deltas > 0)
+        arg_inter = np.argmax(deltas >= 0)
```

Comment:
Whats the reason for including =0?

Reply:
This needs some name updating, as the point being used is not always the magnetic centre of the plasma, in which case you might need the '='.  Updated the name and comments.